packages/helm/symphony/values.yaml

```diff
@@ -27,6 +27,11 @@ redis:
   asSidecar: false
   image: redis/redis-stack-server:latest
   port: 6379
+  persistentVolume:
+    enabled: false
+    storageclass: 
+    accessMode: ReadWriteOnce
+    size: 1Gi
```

Comment:
If customer wants to change the size in upgrade, will this create a new PV or scale up/down the existing PV?

Reply:
Kubernetes only support expanding volume dynamically provisioned when storage class supports it.